hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/acl/OMKeyAclRequest.java

```diff
@@ -146,6 +148,30 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
    */
   abstract String getPath();
 
+  public OmBucketInfo getBucketInfo(OzoneManager ozoneManager) {
+    OmBucketInfo buckInfo = null;
+    try {
+      ObjectParser objectParser = new ObjectParser(getPath(),
+          OzoneManagerProtocolProtos.OzoneObj.ObjectType.KEY);
+
+      String volume = objectParser.getVolume();
+      String bucket = objectParser.getBucket();
+
+      String buckKey =
+          ozoneManager.getMetadataManager().getBucketKey(volume, bucket);
+
+      try {
+        buckInfo =
+            ozoneManager.getMetadataManager().getBucketTable().get(buckKey);
+      } catch (IOException e) {
+        e.printStackTrace();
```

Comment:
Please add log message and remove e.printStackTrace() statement.

Reply:
Done